app/src/main/java/nl/tudelft/b_b_w/model/BlockFactory.java

```diff
@@ -21,16 +21,17 @@
      * @param _ownHash            Hash value of the block
      * @param _previousHashChain  Previous hash value of the block in the blockchain
      * @param _previousHashSender Previous hash value of the sender of the block
-     * @param _publicKey          Public key of the block
+     * @param _publicKey          Public key of the sender
+     * @param _iban               IBAN number of a contact
      * @return Returns a new block
      * @throws IllegalArgumentException when the input string type is invalid
      */
-    public static Block getBlock(String type, String _owner, int _sequenceNumber, String _ownHash, String _previousHashChain, String _previousHashSender, String _publicKey) throws IllegalArgumentException {
+    public static Block getBlock(String type, String _owner, int _sequenceNumber, String _ownHash, String _previousHashChain, String _previousHashSender, String _publicKey, String _iban) throws IllegalArgumentException {
         switch (type) {
```

Comment:
Since you are editing the block attributes now already, you can easily fix this issue too.